bucket/win-dynamic-desktop.json

```diff
@@ -0,0 +1,40 @@
+{
+    "version": "3.3.0",
+    "homepage": "https://github.com/t1m0thyj/WinDynamicDesktop",
+    "description": "Dynamic Desktop.",
```

Comment:
I have no idea, what does this app do.

Reply:
How about `Cycle through multiple wallpapers based on the times of sunrise and sunset.` ?

Reply:
I do not know, why are you asking, when you still push it without answer to your question.